x-pack/plugins/observability/server/assets/constants.ts

```diff
@@ -5,8 +5,8 @@
  * 2.0.
  */
 
-export const SLO_RESOURCES_VERSION = 2;
-export const SLO_SUMMARY_TRANSFORMS_VERSION = 3;
+export const SLO_RESOURCES_VERSION = 3;
+export const SLO_SUMMARY_TRANSFORMS_VERSION = 4;
 
 export const SLO_COMPONENT_TEMPLATE_MAPPINGS_NAME = '.slo-observability.sli-mappings';
```

Comment:
I think we can change the mappings to be text or another type instead of keywords